test/request/request.test.ts

```diff
@@ -139,10 +139,27 @@ describe('Test request header', () => {
     let request = new Request(event)
 
     expect(() => {
-      request.header('abc')
+      request.headerOrFail('abc')
     }).toThrow(/key not found/)
   })
+  
+  test('getOrFail retrieve header', () => {
+    let event = getEvent()
+    event.headers['test'] = 'val'
+    let request = new Request(event)
+    expect(request.headerOrFail('test')).toBe('val')
+  })
 
+  
+  test('undefined header value', () => {
```

Comment:
We need a test that tests that an error is thrown for the new method

Reply:
at line 136, if I'm not mistaken